doc/ssh_config.md

```diff
@@ -0,0 +1,230 @@
+| ports          |     Port number for SSH                                                   |     1-65535             |     22           |
+| tcp-forwarding |     enable the option of tcp forwarding for   the ssh server              |     enabled/disabled    |     enabled      |
+| x11-forwarding |     enable the option of x11 forwarding for   the ssh server              |     enabled/disabled    |     enabled      |
+
+###  1.8. <a name='Init flow'></a>Init flow 
+
+During init flow we will set default ssh policies, same as default values in DebianOS. Default values will be added to init_cfg.json.j2, and updated in sshd_config file accordingly.
+####  1.8.1. <a name='FeatureDefault'></a>Feature Default
+
+Description of default values in init_cfg.json regarding SSH global config:
+```
+login attempts: 6 
+login timeout: 120 //seconds
+ports: 22
+tcp-fowarding: Enabled
+X11-forwarding: Enabled
+```
+###  1.9. <a name='SAI api'></a>SAI api
+NA
+###  1.10. <a name='Configurationandmanagement'></a>Configuration and management 
+
+####  1.10.1. <a name='SSH_GLOBALconfigDBtable'></a>SSH_GLOBAL configDB table
+
+```
+SSH_GLOBAL:{
+	policies:{
+		"login-attempts": {{num}}
+		"login-timeout": {{secs}}
+		"ports": {{num}}
+		"tcp-forwarding": {{True/False}}
+		"x11-forwarding": {{True/False}}
+	}
+}
+```
+#### 1.10.2. ConfigDB schemas
+```
+; Defines schema for SSH_GLOBAL configuration attributes in SSH_GLOBAL table:
+key                                   = "POLICIES"             ;ssh global configuration
+; field                               = value
+LOGIN_ATTEMPTS                        = 3*DIGIT                 ; number of login attepmts, should be 100 max
+LOGIN_TIMEOUT                         = 3*DIGIT                 ; login timeout in secs unit, max is 600 secs
+PORTS                                 = 5*DIGIT                 ; ssh port number - max is 65535
+TCP_FORWARDING                        = "True" / "False"
+X11_FORWARDING                        = "True" / "False"
+```
+
+####  1.10.3. <a name='CLIYANGmodelEnhancements'></a>CLI/YANG model Enhancements
+```yang
+//filename:  sonic-ssh_global.yang
+module sonic-sshg {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-ssh_global";
+	prefix sshg;
+
+    description "SSH GLOBAL CONFIG YANG Module for SONiC OS";
+
+	revision 2022-08-29 {
+        description
+            "First Revision";
+    }
+
+    container sonic-ssh_global {
+		container SSH_GLOBAL {
+			description "SSH GLOBAL CONFIG part of config_db.json";
+			container POLICIES {
+				leaf login_attempts {
+					description "number of login attepmts";
+					default 6;
+					type uint8 {
+						range 1..100;
+					}
+				}
+				leaf login_timeout {
+					description "login timeout (secs unit)";
+					default 120;
+					type uint32 {
+						range 1..600;
+					}
+				}
+				leaf ports {
+					description "ssh port number";
+					default 22;
+					type uint32 {
+						range 1..65535;

```

Comment:
sshd support bind to multiple ports, do you want to support?

Reply:
At this current stage we are going to support only 1 port binding.
If users will want to bind multiple ports - they'll have to change the config manually.

Reply:
The leaf is named as `ports`. Could you support multiple ports? Is it requiring significant extra work?

Reply:
The comment is still applicable. @liat-grozovik for vis.

Reply:
We want to support multiple ports